linux_os/guide/system/selinux/selinux-booleans/sebool_secure_mode_insmod/ansible/shared.yml

```diff
@@ -0,0 +1,29 @@
+# platform = multi_platform_ol
+# reboot = false
+# strategy = enable
+# complexity = low
+# disruption = low
+
+- (xccdf-var var_secure_mode_insmod)
+
+- name: {{{rule_id }}} - Ensure libsemanage-python installed
+  package:
+    name: libsemanage-python
+    state: present
+
+- name: {{{rule_id }}} - Set SELinux boolean secure_mode_insmod accordingly
+  seboolean:
+    name: secure_mode_insmod
+    state: "{{ var_secure_mode_insmod }}"
+    persistent: yes
+
+# Preload vfat in initramfs, otherwise the system fails to reboot
+- name: {{{rule_id }}} - Add vfat to static kernel module list
+  lineinfile:
+      create: yes
+      dest: /etc/modules-load.d/vfat.conf
+      regexp: ^\s*\bvfat\b
+      line: "vfat"
+
+- name: {{{rule_id }}} - Regenerate initramfs
+  shell: dracut -f --regenerate-all
```

Comment:
The `command` module should be sufficient here instead of `shell` module. Could you update, please?